upstairs/src/lib.rs

```diff
@@ -5162,7 +5380,8 @@ impl Upstairs {
         // Mark this downstairs as bad if this was a write or flush
         if let Err(err) = ds.client_error(ds_id, client_id) {
             if err == CrucibleError::UpstairsInactive {
-                println!(
+                info!(
```

Comment:
I could go `warn` or `error` here (but yeah, not `info`).  If the upstairs has become inactive, then this could be part of the tear down which might not be an error.  I'll make it error though, and if it is in the context of a deactivate we will probably know and will ignore.